### ISSUE TITLE
Tireless and general feedback

### ISSUE THREAD

**Issue by ghost** [YourWaifuIsShip/Azur-Lane-Guides#35]
Cheers, been using your and similar guides for almost a year, this game's pretty damn complicated and doesn't really require strong mechanics knowledge, so it's not something everyone has.

I'll be brief.  Definitely preferred the tiered approach.  It's your own damn fault if you just fill up on high tiers without added thought and do badly.  Rather, the number is a purely subjective and arbitrary rank saying if you should invest in a new character.  I admittedly find that number useful, as someone who critically lacks a sense of grounded basis when comparing characters due to lack of game knowledge and the fact that there are so damn many.

Of course, the roles info helps remove some of that arbitrary nature by adding context, which is also appreciated.  But your guide, you do you.  Many thanks.

**Comment by YourWaifuIsShip** [comment]
Alright, noted - as mentioned elsewhere, I'm currently trying the 'roles chart' model as a way to dodge the tiers and avoid unnecessary sperging over pointless and arbitrary numbers. 
On paper it should prove to be both informative and a good enough way to tell at a glance what X does and how well it does it - that's the hope, anyway~

**Comment by H2050** [comment]
People come for a tiered list. If you remove all the tiers, you list will become obselete.

**Comment by Fishman465** [comment]
His goal is for people to read the f'ing descriptions/roles and not just the tier numbers

**Comment by YourWaifuIsShip** [comment]
If all they are interested in is an arbitrary, pointless numbers that does not in any significant way represent the unit's worth and its capabilities, they are free to use the extant tierlists - clearly they were not the sort to read the descriptions, and the Guide in various places (not least the Readme) makes it **very** clear it is not meant to be a simple "this good" / "this bad" list but an annotated compendium of the various relevant units.

tldr;
The descriptions are far more important than the tiers. If all you want are the tiers, go to a tierlist; if you want to understand why a unit is good or not, stay here.
Either works for me.

**Comment by Enferlain** [comment]
You give too much credit to people who want their favorite characters justified by numbers given to them by other people.

**Comment by Bu11etmagnet** [comment]
I liked the tiered version better. At some point you'll get several ships with similar roles (e.g. barrager BBs) and it was nice to have an indicator that "this ship does X better than that other ship".

**Comment by H2050** [comment]
I shouldn't say the list is obsolete, it will still be useful to many including myself.
I have used the list for a long time, I have grown to know what ship is good and bad based on my knowledge that was built from this list.
Now however, if I was a beginner and come to find your list, I wouldn't have a clue on deciding if a ship with "Good Barrage" is better than one with "Great DPS with a down side" or not. This list is most useful to beginners, and you had just made it difficult for them.

I have a suggestion, instead of tiered, you assign simple recommendations like "Get" and "Must Get". That way you highlight important while not discriminating weaker ones.
